collector_test.go

```
@@ -0,0 +1,941 @@
+		t.Fatalf("expected require_ssl normalization, got %#v", record.Input.Config["ssl_enforcement"])
+	}
+	if _, ok := record.Labels["primary"]; ok {
+		t.Fatal("raw payload hash leaked into labels")
+	}
+	if len(record.Input.Collection.RawPayloadHashes) == 0 {
+		t.Fatal("expected raw payload hash under collection.raw_payload_hashes")
+	}
+}
+
+func TestCollectorAppliesAPITimeoutToTargetResolution(t *testing.T) {
+	cfg, err := parsePluginConfig(map[string]string{
```

golangci-lint (2.12.2)

[error] ineffectual assignment to cfg

(ineffassign)